pkgs/desktops/gnome-3/extensions/impatience.nix

```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   name = "gnome-shell-impatience-${version}";
-  version = "6564c21e4caf4a6bc5fe2bf21116d7c15408d494";
+  version = "version-0.4.5";
```

Comment:
This looks like one "version" to many :-) (If not, please add the extra version elsewhere, so the package name does not include the "version" string.)